lib/express_validator.js

```diff
@@ -174,7 +174,9 @@ var expressValidator = function(options) {
     req.asyncValidationErrors = function(mapped) {
       return new Promise(function(resolve, reject) {
         var promises = req._asyncValidationErrors;
-        Promise.settle(promises).then(function(results) {
+        Promise.all(promises.map(function(promise) {
```

Comment:
Since we had to move from using a well-documented method to now using their recommended fix, can you add a comment above this line just explaining what this chunk of code does?